### ISSUE TITLE
install fails when unzip creates axiom-master

### ISSUE THREAD

**Issue by rsubasic** [dlwicksell/axiom#2]
Nice work.  Thanks!

The distributed Zip file, "axiom-master.zip" is created inside axiom-master directory.  As a result, when you run the install script, it fails on this section:

cd `find ~/ -type d -name axiom -print` #need to install from the repository

The cd command then lands you in your home directory and more things fail.

Solution(s):
1. Create the zip file in "axiom" directory rather than "axiom-master" directory
2. Have the install script confirm if it is already in the correct directory to avoid the cd command and the need to change directory.

**Comment by dlwicksell** [comment]
Thank you for bringing this to my attention. I have never tested installation from the ZIP file. Indeed, I never created a ZIP file, Github did, and it was named axiom-master.zip rather than axiom.zip. If it had been named axiom.zip, the find command would have worked fine, finding the directory contained in the ZIP file.

I have always cloned the repository with git instead. That will create the repo in the axiom directory, where the install script should work correctly. So I'd suggest cloning the repo for now, and I'll fix the install script so that it works regardless of the ZIP file name. You could also just change the name of the directory from axiom-master to axiom after unzipping the ZIP file, and it should work correctly.

Thanks again.

**Comment by rsubasic** [comment]
I got the install working before. Still thanks for the advice. In the README.md file, you give an address from which to do a clone:

hg clone https://bitbucket.org/dlw/axiom

But the link is broken. All the best!


Raymond M. Subasic

On Sep 29, 2020, at 5:21 PM, David Wicksell <notifications@github.com> wrote:

Thank you for bringing this to my attention. I have never tested installation from the ZIP file. Indeed, I never created a ZIP file, Github did, and it was named axiom-master.zip rather than axiom.zip. If it had been named axiom.zip, the find command would have worked fine, finding the directory contained in the ZIP file.

I have always cloned the repository with git instead. So I'd suggest cloning the repo for now, and I'll fix the install script so that it works regardless of the ZIP file name. You could also just change the name of the directory from axiom-master to axiom after unzipping the ZIP file, and it should work correctly.

Thanks again.

**Comment by dlwicksell** [comment]
Yeah, this repo was originally hosted on bitbucket, and was using mercurial, rather than git. That was a long time ago. I didn't do a very thorough job of updating the documentation when I moved it from bitbucket to github, and from mercurial to git. Sorry about that.

When i fix the issue with the install script, I'll update all the documentation as well. So I'll leave this issue open until I push out a new commit with that done. Thank you.

**Comment by rsubasic** [comment]
Many thanks David. Thought you should know what I found. 

Raymond M. Subasic

On Sep 29, 2020, at 5:39 PM, David Wicksell <notifications@github.com> wrote:

﻿
Yeah, this repo was originally hosted on bitbucket, and was using mercurial, rather than git. That was a long time ago. I didn't do a very thorough job of updating the documentation when I moved it from bitbucket to github, and from mercurial to git. Sorry about that.

When i fix the issue with the install script, I'll update all the documentation as well. So I'll leave this issue open until I push out a new commit with that done. Thank you.
